### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by jvies**
hi all,

I've just discovered Elkarbackup a few days ago. It matches my expectation for a web based GUI around RSYNC (that I use since years for my backups).

Here is a PR to add the French translation. It think it needs to be reviewed by another french speaker if you have one around here.

### CODE REVIEW

**Comment by elacunza** [comment]
Thanks for your contribution! Let's see whether @xezpeleta can review the translation, otherwise we can merge and fix later if necessary.

**Comment by xezpeleta** [comment]
Bonjour Jérémy,

Nous allons jeter un oeil et nous vous tiendrons au courant

Merci beaucoup!

**Comment by jvies** [comment]
Bonjour,

j'ai laissé "backup" et pas "sauvegarde" partout.

N'hésitez pas à me faire des remarques, je peux y consacrer quelques heures
à l'occasion !

File: translations/BinovoElkarBackup.fr.yml [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+ <h4>Trier les tâches</h4>
+ Vous pouvez changer l'ordre d'exécution des tâches. C'est très utile si vous avez plusieurs tâches planifiées au même horaire.
+ <p>Vous pouvez réorganisez à votre guise en drag & drop-ant</p>
+preferences_help: |
+ <h4>Gestion des préférences utilisateur</h4>
+ Tout ce que vous changez ici n'affectera pas les autres utilisateurs.
+ <h4>Langue</h4>
+ Choisissez votre langue préférée
+ <h4>Entrées par page</h4>
+ Choisissze le nombre de lignes que vous voulez afficher par page
```
Comment (xezpeleta) [inline translations/BinovoElkarBackup.fr.yml:473]: Choisissez :)

File: templates/default/client.html.twig [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
             <div class="panel panel-default">
               <div class="panel-heading panel-collapsable-heading">
-                <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">Advanced</a></div>
+                <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">{% trans %}Advanced{% endtrans %}</a></div>
```
Comment (xezpeleta) [inline templates/default/client.html.twig:52]: Hi! This correction is not related to the french translation.
Could you use a separate PR for these changes? Thanks!

File: templates/default/job.html.twig [hunk 3]
```diff
@@ -62,7 +62,7 @@
                 {{ form_row(form.backupLocation) }}
                 <div class="panel panel-default">
                   <div class="panel-heading panel-collapsable-heading">
-                    <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">Advanced</a></div>
+                    <a class="collapsed" href="#viewadvanced" data-toggle="collapse" data-target="#viewadvanced">{% trans %}Advanced{% endtrans %}</a></div>
```
Comment (xezpeleta) [inline templates/default/job.html.twig:65]: Same here. I think it's better to keep this changes in a separate PR

File: translations/BinovoElkarBackup.en.yml [hunk 4]
```diff
@@ -289,7 +289,7 @@ policy_help: |
  Snapshots once a week.
  <h4>Monthly</h4>
  Snapshots once a month.
- <h4>Monthly</h4>
+ <h4>Yearly</h4>
```
Comment (xezpeleta) [inline translations/BinovoElkarBackup.en.yml:292]: Fixed English translations. Can I ask you a separate PR? Thanks

**Review by xezpeleta — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. You did a great job.

I think we should keep this PR only for french translations. It would be nice to have the other changes (english corrections etc) in a separate PR.

Merci beaucoup!!!

**Review by xezpeleta — APPROVED** [review]

**Comment by xezpeleta** [comment]
> Bonjour, j'ai laissé "backup" et pas "sauvegarde" partout. N'hésitez pas à me faire des remarques, je peux y consacrer quelques heures à l'occasion !

Parfait! Nous avons fait exactement la même chose en basque et en espagnol.

Merci beaucoup encore

**Pull request merged**